networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/BeaconChainMethodsTest.java

```diff
@@ -93,35 +93,30 @@ public void shouldCreateVersionedBlocksByRangeMethodWithAltairEnabled() {
     final BeaconChainMethods methods = getMethods(TestSpecFactory.createMinimalAltair());
 
     assertThat(methods.beaconBlocksByRange().getIds())
-        .containsExactly(
-            "/eth2/beacon_chain/req/beacon_blocks_by_range/2/ssz_snappy",
-            "/eth2/beacon_chain/req/beacon_blocks_by_range/1/ssz_snappy");
+        .containsExactly("/eth2/beacon_chain/req/beacon_blocks_by_range/2/ssz_snappy");
   }
 
   @Test
   public void shouldCreateUnversionedBlocksByRangeMethodWithAltairDisabled() {
     final BeaconChainMethods methods = getMethods();
 
     assertThat(methods.beaconBlocksByRange().getIds())
-        .containsExactly("/eth2/beacon_chain/req/beacon_blocks_by_range/1/ssz_snappy");
+        .containsExactly("/eth2/beacon_chain/req/beacon_blocks_by_range/2/ssz_snappy");
   }
 
   @Test
   public void shouldCreateVersionedBlocksByRootMethodWithAltairEnabled() {
     final BeaconChainMethods methods = getMethods(TestSpecFactory.createMinimalAltair());
 
     assertThat(methods.beaconBlocksByRoot().getIds())
-        .containsExactly(
-            "/eth2/beacon_chain/req/beacon_blocks_by_root/2/ssz_snappy",
-            "/eth2/beacon_chain/req/beacon_blocks_by_root/1/ssz_snappy");
+        .containsExactly("/eth2/beacon_chain/req/beacon_blocks_by_root/2/ssz_snappy");
   }
 
   @Test
-  public void shouldCreateUnversionedBlocksByRootMethodWithAltairDisabled() {
+  public void shouldCreateUnversionedBlocksByRootMethodWithAltairDisabled() { // TODO remove test?
```

Comment:
basically before it was testing a call to v1 not v2 was made, v1 doesnt exist (and so we can't call it...)

by that logic it's fair to say delete test

The main unanswered question is probably if we want to see that calling a function that doesn't exist results in an empty output, rather than something like things going boom...
Can we call it in this way any more? It does sound like we just won't be able to use phase0 at all without these functions? If that's the case, I think i'm happy with deletion, just wondering if there's any case where we need to actually create phase0...